workspaces/linguist/plugins/linguist-backend/package.json

```diff
@@ -41,6 +42,7 @@
   "dependencies": {
     "@backstage-community/plugin-linguist-common": "workspace:^",
     "@backstage/backend-defaults": "^0.5.2",
+    "@backstage/backend-openapi-utils": "^0.3.0",
```

Comment:
May not be needed here yet, but just a heads up - the latest release added better support for recursive types and less Typescript inference. https://github.com/backstage/backstage/pull/23610.